browser/src/view/components/Content/ContentCollection.jsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import './ContentCollection.scss';
+import { Skeleton } from '../Skeleton/Skeleton';
+
+export const ContentCollection = () => {
+  return (
+    <table>
+      <colgroup>
+        <col className='accounts'></col>
+        <col className='description'></col>
+        <col className='date'></col>
+        <col className='labels'></col>
+        <col className='status'></col>
+      </colgroup>
+      <thead>
+        <tr>
+          <th>Accounts</th>
+          <th>Description</th>
+          <th>Date</th>
+          <th>Labels</th>
+          <th>Status</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
```

Comment:
You may want to break the skeleton table body content out into its own component just to easily switch out the components after things have been fetched.